repairnator/repairnator-dockerpool/src/main/java/fr/inria/spirals/repairnator/dockerpool/Launcher.java

```diff
@@ -88,6 +91,14 @@ private JSAP defineArgs() throws JSAPException {
         // --pushurl
         jsap.registerParameter(LauncherUtils.defineArgPushUrl());
 
+        FlaggedOption opt2 = new FlaggedOption("repairTools");
+        opt2.setLongFlag("repairTools");
+        opt2.setList(true);
+        opt2.setListSeparator(',');
+        opt2.setHelp("Specify one or several repair tools to use separated by commas (available tools might depend of your docker image)");
+        opt2.setRequired(true);
+        jsap.registerParameter(opt2);
+
```

Comment:
This should be defined in `LauncherUtils` because we also have this in pipeline launcher.

Reply:
We have this in pipeline but not exactly on the same form: in pipeline I can discover what Repair tools I have, not here because I'll use the docker image. That's why I decided to not put it in LauncherUtils.